site/docs/master/sonobuoy-config.md

```diff
@@ -12,87 +12,75 @@ and you will see the default configuration. Below is a description of each of th
-        - Default is empty string
-        - To get logs from all namespaces use ".*"
-    - **SonobuoyNamespace**: bool
-        - If set to true, get pod logs from the namespace Sonobuoy is running in. Can be set along with a `Namespaces` field or on its own.
-        - Default value is true
-    - **FieldSelectors**: []string
-        - A list of field selectors, with OR logic. 
-        - For example, to get logs from two specified namespaces
-        `FieldSelectors = ["metadata.namespace=default","metadata.namespace=heptio-sonobuoy"]` 
-        - Each field selector contains one or more chained operators, with AND logic
-        - For example, to get logs from a specified pod 
-        `FieldSelectors = ["metadata.namespace=default,metadata.name=pod1"]` 
-        - Each field selector follows the same format as
-        `k8s.io/apimachinery/pkg/apis/meta/v1/types/ListOptions/FieldSelector`
-        - Can be set along with a Namespaces/SonobuoyNamespace field or on its own.
-    - **LabelSelector**: string
-        - Filtering candidate pods by their labels
-        - Using the same format as
-        `k8s.io/apimachinery/pkg/apis/meta/v1/types/ListOptions/LabelSelector`
-        - For example:
-        `LabelSelector = "app=nginx,layer in (frontend, backend)"`
-        - When set together with other fields, the scope of pods is defined by:
+`Resources`: A list of resources which Sonobuoy will query for in every namespace in which it runs queries. In the namespace in which Sonobuoy is running, `PodLogs`, `Events`, and `HorizontalPodAutoscalers` are also added.
+
+`Filters`: Options for filtering which resources queries should be run against.
+
+ * `Namespace`: A regexp which specifies which namespaces to run queries against.
+ * `LabelSelector`: A Kubernetes [label selector][labelselector] which will be added to every query run.
+
+`Limits`: Options for limiting the scope of response.
+
+ * `PodLogs`: limits the scope when getting logs from pods. The supported parameters are:
+
+    * `Namespaces`: string
+
+        * A regular expression for the targeted namespaces.
+        * Default is empty string
+        * To get logs from all namespaces use ".*"
+    * `SonobuoyNamespace`: bool
+
+        * If set to true, get pod logs from the namespace Sonobuoy is running in. Can be set along with a `Namespaces` field or on its own.
+        * Default value is true
+    * `FieldSelectors`: []string
+
+        * A list of field selectors, with OR logic.
+          For example, to get logs from two specified namespaces `FieldSelectors = ["metadata.namespace=default","metadata.namespace=heptio-sonobuoy"]`
+        * Each field selector contains one or more chained operators, with AND logic
+          For example, to get logs from a specified pod `FieldSelectors = ["metadata.namespace=default,metadata.name=pod1"]`
+        * Each field selector follows the same format as a [Kubernetes Field Selector][fieldselector].
```

Comment:
I opted to link to the Kubernetes docs here which show some examples of field selectors rather than just mentioning the package path.